src/System.Threading/src/System/Threading/ReaderWriterLockSlim.cs

```diff
@@ -55,12 +55,12 @@ public class ReaderWriterLockSlim : IDisposable
         //Specifying if locked can be reacquired recursively.
         private bool _fIsReentrant;
 
-        // Lock specifiation for myLock:  This lock protects exactly the local fields associted
-        // instance of ReaderWriterLockSlim.  It does NOT protect the memory associted with the
+        // Lock specifiation for myLock:  This lock protects exactly the local fields associated
```

Comment:
:bulb: specifi*c*ation

:thought_balloon: Honestly, it seems everything before the `:` could be omitted.